packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Large.Example.tsx

```diff
@@ -1,37 +1,33 @@
-import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import * as React from 'react';
 
-export class PanelLargeExample extends React.Component<
-  {},
-  {
-    showPanel: boolean;
-  }
-> {
-  constructor(props: {}) {
-    super(props);
-    this.state = { showPanel: false };
-  }
+interface IState {
+  showPanel: boolean;
+}
+
+export class PanelLargeExample extends React.Component<{}, IState> {
+  // tslint:disable-next-line:member-access
+  state = {
+    showPanel: false
+  };
 
-  public render(): JSX.Element {
+  public render() {
     return (
       <div>
-        <DefaultButton
-          secondaryText="Opens the Sample Panel"
-          // tslint:disable-next-line:jsx-no-lambda
-          onClick={() => this.setState({ showPanel: true })}
-          text="Open Panel"
-        />
-        <Panel
-          isOpen={this.state.showPanel}
-          // tslint:disable-next-line:jsx-no-lambda
-          onDismiss={() => this.setState({ showPanel: false })}
-          type={PanelType.large}
-          headerText="Large Panel"
-        >
+        <DefaultButton secondaryText="Opens the Sample Panel" onClick={this._onShowPanel} text="Open Panel" />
+        <Panel isOpen={this.state.showPanel} onDismiss={this._onDismissPanel} type={PanelType.large} headerText="Large Panel">
           <span>Content goes here.</span>
         </Panel>
       </div>
     );
   }
+
+  private _onDismissPanel = () => {
+    this.setState({ showPanel: false });
+  };
+
+  private _onShowPanel = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
👍

Reply:
These should be `_showPanel` and `_hidePanel` right?

Reply:
Do we still not want these to be `_showPanel` and `_hidePanel` like in the other examples?

Reply:
Great catch, just updated and pushed.  I think this illustrates how tedious function names are in general.   I'm slapping myself on the wrist right now because I was attempting to consistently name them and then myself inconsistently named then?!

Reply:
Haha no problem, thanks for updating!